policyengine_us/variables/gov/states/nj/tax/income/exclusions/nj_other_retirement_special_exclusion.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class nj_other_retirement_special_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Jersey Other Retirement Special Exclusion"
+    unit = USD
+    documentation = "New Jersey other retirement special exclusion"
+    definition_period = YEAR
+    reference = (
+        "https://www.state.nj.us/treasury/taxation/pdf/current/1040i.pdf#page=21",
+        "https://law.justia.com/codes/new-jersey/2022/title-54a/section-54a-6-15/",
+    )
+    defined_for = StateCode.NJ
+
+    def formula(tax_unit, period, parameters):
+        # If filer (and spouse, if joint) are never eligible for social security benefits, they are eligible for special exclusion.
+        # We will infer that if above age 70 with no social seurity benefits, then eligible.
```

Comment:
Move this into `variables/ssa/ss/never_eligible_for_social_security_benefits.py`, so we can separate assumptions from rules.